ibm/service/cdtektonpipeline/resource_ibm_cd_tekton_pipeline_trigger.go

```diff
@@ -67,13 +67,11 @@ func ResourceIBMCdTektonPipelineTrigger() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"name": &schema.Schema{
 							Type:        schema.TypeString,
-							Optional:    true,
```

Comment:
name and type inside a wokrer are only computed?
We see in resourceIBMCdTektonPipelineTriggerMapToWorker still trying to read name and type?

Reply:
I think this is a mistake. When creating a pipeline trigger you can supply a Worker ID, then in the response it will compute the name and type. I will look at fixing this

Reply:
Addressed in the latest commit: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4235/commits/cbbbe65a29246832e3d02675d4a82db6ac52f6bb